From 9ad0cc7792fc1f0e0fc8985ba8810362707b9b20 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 7 Oct 2019 12:25:35 +0000
Subject: [PATCH] OS-7945 kvm needs to be built with retpolines Reviewed by:
 Jerry Jelinek <jerry.jelinek@joyent.com> Reviewed by: Michael Zeller
 <mike.zeller@joyent.com> Approved by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index cee6818..71245cc 100644
--- a/Makefile
+++ b/Makefile
@@ -123,7 +123,9 @@ KERNEL_CFLAGS = \
 #
 ifneq ($(PRIMARY_COMPILER_VER),4)
 KERNEL_CFLAGS += \
-	-fno-shrink-wrap
+	-fno-shrink-wrap \
+	-mindirect-branch=thunk-extern \
+	-mindirect-branch-register
 endif
 
 USER_CFLAGS = \
-- 
2.21.0

